src/modules/mc_rate_control/mc_rate_control_params.c

```diff
@@ -254,7 +254,6 @@ PARAM_DEFINE_FLOAT(MC_YAWRATE_D, 0.0f);
  *
  * @min 0.0
  * @decimal 4
- * @increment 0.01
```

Comment:
why remove?

Reply:
to align it with `MC_ROLLRATE_FF` and `MC_PITCHRATE_FF`

Reply:
sorry.. decimal shows decimal places... increment does what?